docs/tour/adding_capabilities.mdx

```diff
@@ -0,0 +1,232 @@
+---
+sidebar_position: 2
+---
+
+import Tabs from "@theme/Tabs";
+import TabItem from "@theme/TabItem";
+
+# Adding Capabilities
+
+Going from "Hello World" to a full-fledged application requires just identifying what capabilities your application needs, and then adding them to your actor. Applications in wasmCloud have two types of capabilities: external requests that call a function handler in your actor (like HTTP Server and Messaging) and actor requests that call a function handler in a capability provider (like Key-Value Store and Logging). "Hello World" already has the HTTP Server capability, so let's add more features to this application with functional code and more capabilities.
```

Comment:
this is confusing.. it sounds like we have 2 kinds of capabilities, but what we really want to convey here is that, depending on the provider, the capabilities can call our actors or our actors can call the capabilities